tensorly/backend/jax_backend.py

```diff
@@ -58,29 +58,6 @@ def lstsq(a, b):
         x, residuals, _, _ = np.linalg.lstsq(a, b, rcond=None, numpy_resid=True)
         return x, residuals
 
-    def kr(self, matrices, weights=None, mask=None):
-        n_columns = matrices[0].shape[1]
-        n_factors = len(matrices)
-
-        start = ord("a")
-        common_dim = "z"
-        target = "".join(chr(start + i) for i in range(n_factors))
-        source = ",".join(i + common_dim for i in target)
-        operation = source + "->" + target + common_dim
-
-        if weights is not None:
-            matrices = [
-                m if i else m * self.reshape(weights, (1, -1))
-                for i, m in enumerate(matrices)
-            ]
-
-        m = mask.reshape((-1, 1)) if mask is not None else 1
-        return np.einsum(operation, *matrices).reshape((-1, n_columns)) * m
-
-    @staticmethod
-    def logsumexp(tensor, axis=0):
-        return jax.scipy.special.logsumexp(tensor, axis=axis)
```

Comment:
Why did you remove the logsumexp? I believe it was needed by another package downstream.

Reply:
Oops! Missed that resolving a merge conflict.